test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +150,106 @@ def teardown
       "OSAlias=UniversalR\n" \
       "OSManufacturer=Central Logistics GmbH\n"
 
+    # Begin:  Tests on Public internals; probably only for use inside the Singular object.
+
+    def test_clean_hostname_string
+      hostname = '        microsoft     '
```

Comment:
Why isn't this in TestHostnameList?

Reply:
Sorry.  GA?  

Here are items I've got:
1.  Looks like successful install tests on CentOS 7, Ubuntu 16.04, and OpenSuSE.  
2.  I know I did have a process fallaway overnight before I started running the sequence:
sudo su -l omsagent -c '/opt/microsoft/omsconfig/Scripts/OMS_MetaConfigHelper.py --disable'
echo "trace result of metaconfig $?"
sudo su - omsagent -c '/opt/microsoft/omsconfig/Scripts/GetDscLocalConfigurationManager.py | grep RefreshMode'
---snip---
3.  It looks like this probably stays as I seem to be turning off the update from Dsc, but I only checked it once, not exhaustively, and the 36 hour test needs about 34 more hours at least.

Reply:
GA: general availability. Our GA bundles are any which are labelled GA on our releases page: https://github.com/Microsoft/OMS-Agent-for-Linux/releases

Reply:
it looks like my centos install got an update somehow to its oms_common.rb under opt.  Perhaps I didn't execute the disable fast enough.  I'll try again.

Reply:
D'oh.  I forgot my install script stops all threads at the end of each test.  I need to administer a start to engage the 36 hour test.  It was probably fine all along.